app-frontend/packs/components/baseComponents/LoaderPage.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+
+    <div class="container-loader">
+        <div class="container-loader__loader">
+            <span class="loader__moving-indicator"></span>
+        </div>
+    </div>
+
+</template>
+
+<script>
+export default {
+    name: "LoaderPage"
+}
+</script>
+
+<style scoped>
+
+    .container-loader {
+        position: relative;
+        width: 100%;
+        height: 100%;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+    }
+
+    .container-loader .container-loader__loader {
+        position: relative;
+        width: 200px;
+        height: 200px;
+        border-radius: 50%;
+        background-color: #0D2323;
+        animation: animateLoader 1s linear infinite;
+    }
+
+    .container-loader .container-loader__loader::before {
+        content: "";
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 50%;
+        height: 100%;
+        background: linear-gradient(to top, transparent, rgba(239, 50, 72, 0.8));
+        background-size: 100px 180px;
+        background-repeat: no-repeat;
+        border-top-left-radius: 100px;
+        border-bottom-left-radius: 100px;
+    }
+
+    .container-loader .container-loader__loader::after {
+        content: "";
+        position: absolute;
+        top: 0;
+        left: 50%;
+        transform: translateX(-50%);
+        width: 20px;
+        height: 20px;
+        border-radius: 50%;
+        z-index: 10;
+        background: #fc3747;
```

Comment:
background проперти включает в себя очень много css проперти. По сути у тебя билдится такой css - https://tppr.me/vRDln

Но тебе тут только цвет нужно указать

```suggestion
        background-color: #FC3747;
```

Reply:
исправил